.typescript-declarations/St.d.ts

```diff
@@ -16,6 +15,14 @@ declare namespace imports.gi.St {
 		show(): void;
 		hide(): void;
 		style: string;
+		connect(signal: 'style-changed' | 'popup-menu', callback: (actor: this) => boolean | void): number;
+	}
+
+	type WidgetType = WidgetWithoutParent & Clutter.Actor;
```

Comment:
Please add a comment here why this is necessary so everyone knows, like:

"this is the only way we can extend a class when its bases has different signatures` or something like this. Probably this is not the best explanation

Reply:
I think your comment is okay. I just added additional also link to this pr.